Doc/reference/ldap-modlist.rst

```diff
@@ -19,6 +19,15 @@ The :mod:`ldap.modlist` module defines the following functions:
    names which shall be ignored completely. Attributes of these types will not appear
    in the result at all.
 
+   The value parts of the *entry* dictionary must be any of a list of
+   :py:class:`bytes` objects, a :py:class:`bytes` object, or :py:const:`None` (in Python 3).
+   The :py:class:`bytes` object is treated as a list which contains
+   single :py:class:`bytes` object.
+   These :py:class:`bytes` data are passed to the C interface without
+   any conversion.
+   :py:const:`None` means :c:data:`NULL` in C (no values).
```

Comment:
That's not correct; addModlist doesn't accept `None` as a value.
Could you show how you tested this?

Reply:
Oh, sorry. I looked only `Tuple_to_LDAPMod()` in Modules/LDAPObject.c .

You might be able to eliminate the code below.
https://github.com/python-ldap/python-ldap/blob/master/Modules/LDAPObject.c#L142-L144

Reply:
I wrote a test on `None` case.
Please tell me if you don't want to include it in your test suites.